packages/chain-helpers/src/blockchainApiConnection/__mocks__/BlockchainApiConnection.ts

```diff
@@ -254,8 +258,11 @@ const __mocked_api: any = {
       // default return value decodes to BN(0)
       // default return value decodes to AccountInfo with all entries holding BN(0)
       account: jest.fn(
-        async (address: IPublicIdentity['address'], cb): Promise<AccountInfo> =>
-          TYPE_REGISTRY.createType('AccountInfo')
+        async (
+          address: IPublicIdentity['address'],
+          cb
+        ): Promise<AccountInfoWithProviders> =>
+          TYPE_REGISTRY.createType('AccountInfoWithProviders')
```

Comment:
As explained below, `AccountInfo` is the type used in Substrate 2.0 but with Substrate 3.0, we have to use `AccountInfoWithProviders`.